Week9/MovieViewer/app/src/test/java/com/wmw/movieviewer/ui/MovieDetailViewModelTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.wmw.movieviewer.ui
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import org.junit.Before
+
+import org.junit.Assert.*
+import org.junit.Rule
+import org.junit.rules.TestRule
+
+class MovieDetailViewModelTest {
```

Comment:
This test doesn't really do anything, you should've ran some methods and asserted/verified behavior!

Reply:
@UnstableDefault additions were because of the warnings in the compilier about experiemental code, but they are real annoying, so I am going to take the out :)